src/test/data/JsonProjectBookStorageTest/invalidProjectProjectBook.json

```diff
@@ -1,5 +1,6 @@
 {
   "projects": [ {
-    "name": "Project with invalid name field: Ha!ns Mu@ster"
+    "name": "Project with invalid name field: Ha!ns Mu@ster",
+    "description": "description"
```

Comment:
I think it would be useful to implement the Json serialization such that if description does not exist, then it is taken to be empty (""), so we don't have to add a description to all the sample data

Reply:
Good suggestion, but could we push this to our next iteration if it is not too urgent? I am still not very familiar with how Json works, will need some time to work on it!